### PR TITLE
Enhance the emulator support docs

### PR DESCRIPTION
**Pull request by SuperGamer18**
Oh well, I decided to take my time and improve the edit even further. As usual, if there's anything you want me to change and stuff like that don't hesitate to tell me ;3
https://github.com/SuperGamer18/docs/blob/updates/docs/general/emulator-support-and-issues.md (I also got the Spanish version on par with the English version but I'm not sure if I should put it here as well or make a separate PR)
Here are some of the changes compared to what was first seen in that PR:
- Revised console and core/emulator names to be more accurate to their RetroAchievements/RetroArch versions (I had to make slight reorders as a consequence)
- More punctuation, use of "the", "a", etc. and slight/moderate rewording of a bunch of phrases.
- Added even more info about certain emulators (eg being divided into X cores, not emulating an add-on, etc.)
- Removed redundant sections for Neo Geo Pocket and WonderSwan and combined them with the main one (as all cores emulate both the black & white AND color models and are not separate platforms on RA)
Apart from that, I'm unsure about two things:
1) Should we remove all info on the Citra cores? (As you can no longer get them in the online updater)
2) What's the Spanish translation for the "Enable Dual Core (speedup)" option that must be disabled in Dolphin? (I can't use Dolphin on either of my devices because RA isn't supported on Android + they're too underpowered anyways) (btw @wescopeland I'm not sure if you know Spanish at all so if you see this you can skip this question)

### CODE REVIEW

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md:1]: I won't leave line comments since there are so many instances, so please take a pass through and if anything is missed, it'll be more reasonable to single them out in subsequent reviews.

1. I think it's important to remove ambiguity from this doc by replacing all instances of ❓ and ⌛ with ❌.  As a reference document intended for players, cores should either be listed as supported or not.  I did not leave individual comments for these instances.

2. Speculative comments aren't appropriate for a player-focused document and should be omitted.  They do not provide useful information to players.  All sub-bullets for unsupported cores should be removed.  Sub-bullets for supported cores should only include useful information to players such as "does not support save states." I did not leave individual comments for these instances.

3. Words like "you" should not be used.  The document should be informative in nature and refer only to the core.

4. Several unsupported cores have a standard header of "❌ _Not supported_ - needs hashing method and memory map"  Recommend replacing with "❌ _Not supported_"  The information after the hyphen is irrelevant to players. I did not leave individual comments for these instances.

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md]: Recommend removing "If you see one..." as the player should understand to open a ticket if an achievement doesn't work.

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md:101]: Recommend adding a standard _Not supported_ top line like other unsupported consoles.

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md]: Remove "You must"

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md]: Recommend omitting Pizza Boy altogether.

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md]: Recommend "Must use version 2407-68 or newer.

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md]: Recommend "Ensure "Enable Dual Core (speedup)" option is unchecked as it is unsupported."

File: docs/general/emulator-support-and-issues.md
Comment (suspect15) [inline docs/general/emulator-support-and-issues.md]: Recommend removing this comment.  Not useful to players.

**Review by suspect15 — COMMENTED** [review]
Updating this doc is a great endeavor and should benefit a lot of players.  In the future, I'd recommend only submitting a PR for the English version and after it is reviewed and merged, then create the Spanish version that way there aren't multiple files to review.

I can only comment on the English version, but the Spanish should be updated as well if these recommendations are accepted.

File: docs/general/emulator-support-and-issues.md
Comment (SuperGamer18) [reply docs/general/emulator-support-and-issues.md]: Which one do you mean? The first one or the second one?

Comment (suspect15) [reply docs/general/emulator-support-and-issues.md]: Both of them.  I don't think anyone assumes you can modify game files and they still will hash properly.  The second comment doesn't actually help players either.

**Comment by SuperGamer18** [comment]
Sorry, I meant to send this yesterday but GitHub was down when I finished it.
https://github.com/SuperGamer18/RetroAchievements-docs/blob/updates/docs/general/emulator-support-and-issues.md
I've taken a few of your suggestions, particularly number 3 and everything that isn't number 1, 2 or 4 (Amiga was also missing the Not Supported line)
For number 1, I would keep them because there might be people who wanna try these cores/emulators and see if they work or not (this might be useful for developers as well), though I have removed/replaced some that I think aren't very likely to be supported and stuff like that.
For number 2, I'm not entirely sure what's speculation and what's not, so I may have left something there (I have removed messages that I think are unlikely to be true or unnecessary).
For number 4, I would also keep those because I don't think this guide is just for players. I think developers would find these comments useful if they wanna make an effort to support any kind of system.
Also I made a question in the original comment that you may wanna respond. 
(It's the one that says the following: Should we remove all info on the Citra cores? (As you can no longer get them in the online updater))

**Comment by suspect15** [comment]
We may be talking past each other with the term developer.  When you say you think certain information would be helpful to developers are you talking about RA achievement developers or potential emulator/core integration developers?

**Comment by SuperGamer18** [comment]
In these cases, I'm talking about both (number 1 refers to cheevo developers and number 4 refers to emulator/core integration devs). Sorry if that wasn't clear enough.

**Comment by suspect15** [comment]
For achievement developers and players alike, the only data they should need from this doc is which cores are supported.  There's no value in players testing cores on their own.  There is a process for validating cores that involves a small group of very technically capable people.  It's not the sort of endeavor that benefits from crowd sourcing.  Players testing cores will generate tickets which will end up wasting developer time.  I also don't see any real value in listing cores that are not supported.

I see the use of this doc being players checking to see which cores RA supports before they begin playing a new system.  Then, if a player encounters an issue using a supported core, they know they should open a ticket.  If a developer gets a ticket and they aren't sure if the player's core is supported they can check the doc.  If the core is supported, they should troubleshoot the ticket, if not they should close it.  I think this is really all the doc should do.

If an integration developer notices a core they are familiar with is not supported, they can inquire with the admin team as to what rigor needs to be undertaken to add compatibility or validate it.

I think all additional information is unnecessary as none of it is actionable.  I think we should strive for docs to be clear, concise and correct over unnecessarily informative.

What specific, actionable value do you see in listing unsupported or untested cores and associated notes?

**Review by wescopeland — CHANGES_REQUESTED** [review]
Would like to wait on @suspect15 or another developer's approval before merging.

**Comment by SuperGamer18** [comment]
Hmmmmmm... You've got a point there. Apart from telling achievement devs which cores should they avoid, I can't really think of anything else... Although the vast majority of this information you want me to remove, etc. was already in the page, I really haven't added too much, so whatever you guys tell me to remove, I'll do it whenever I can.

**Comment by suspect15** [comment]
Speaking as a dev, I only would need to know which cores I could use.  If a core wasn't listed as supported, then I would certainly not assume it was viable.

The page was in dire need of a significant oversimplification and since are interested in cleaning it up, this is a great opportunity to slash and burn all the extra unnecessary info that it had accumulated.  Many of the existing docs will benefit from simplification and while it's a slow process, it's definitely an important undertaking.

**Comment by SuperGamer18** [comment]
Alright, I'll remove all unsupported emulators and remove everything after the *not supported* line, as you've suggested 3 weeks ago. If any emulator/core becomes officially supported, I might update the page to reflect that. Also since only supported emulators will be listed, should I remove the ✅ emoji from the supported emulators or not?

**Comment by SuperGamer18** [comment]
https://github.com/SuperGamer18/RetroAchievements-docs/blob/updates/docs/general/emulator-support-and-issues.md
I finished the simplification, as you told me and to accommodate for that I did a few more minor changes such as removing "this is **the only officially supported** emulator" messages and similar for platforms with only one supported emulator/core and adding punctuation after the *not supported* message. Also I will update the Spanish version with similar changes whenever this gets approved.

**Review by suspect15 — CHANGES_REQUESTED** [review]
The English emulator-support-and-issues doc looks great and is ready to be merged.  The minor update to the set-plagiarism doc is ready as well.  Please update the Spanish version of emulator-support-and-issues to match the English version and the PR will be ready for merging.  Great work on this.

**Comment by SuperGamer18** [comment]
Thank you. I will properly update the Spanish version as soon as I find out the translation for the unsupported option in Dolphin.

**Review by suspect15 — COMMENTED** [review]
Changes look good.  Thank you for the continued efforts.  This PR is ready to be merged.

**Review by suspect15 — APPROVED** [review]
Changes look good.  All docs have been simplified and English and Spanish docs have been aligned.

**Comment by SuperGamer18** [comment]
Unfortunately I don't speak Portuguese so I'll let whoever speaks it natively to update the Portuguese version, using either language (preferably English) as the base. Also there are 2 minor changes I did to both languages: I added the BizHawk cores from the GBC part over to the GB part as it is obvious that they're going to be able to emulate both handhelds and I changed the Pokemon Mini title to reflect how the name is usually spelled in Pokémon media (Pokémon Mini).

File: docs/general/emulator-support-and-issues.md
Comment (wescopeland) [inline docs/general/emulator-support-and-issues.md]: While we're touching this line, may be worth fixing this typo:
```suggestion
  - There may be memory leaks and/or burn-in when using save states.
```

Comment (SuperGamer18) [reply docs/general/emulator-support-and-issues.md]: Fixed. I didn't really notice this before, so it's either one of my accidents or it was already like that before my edits.

**Comment by SuperGamer18** [comment]
In case you ask about that last commit, that was me fixing subsets.md, I got inspired while looking at issue #94.

**Comment by wescopeland** [comment]
Any chance we can get that into a separate PR? PRs work best (and are reviewed faster) when changes are broken up logically by subject matter.

**Comment by SuperGamer18** [comment]
I've tried, but for some reason GitHub isn't letting me do that. When I compare the main repo with my *updates* branch the following appears:
Screenshot
(Yes, I'm using an old ass device, but does that really matter here?)

**Comment by wescopeland** [comment]
@SuperGamer18 Would you be willing to revert the changes to the subset page? I was actually about to merge this PR right before you made them. Once this PR is merged, let's open those in a separate PR.

After reverting, you'll always be able to reference those changes at this URL: https://github.com/RetroAchievements/docs/pull/87/commits/89d8820004c15a85fc9e46dc4b9a71da7476a673

**Comment by SuperGamer18** [comment]
Alright, I'll do it. I almost forgot that this PR was open (it was open for pretty much the entire month), sorry. In the meanwhile, you may wanna answer my question in issue #94.

**Comment by SuperGamer18** [comment]
Done, maybe not in the way you expected but anyways...

File: docs/guidelines/content/subsets.md
Comment (wescopeland) [inline docs/guidelines/content/subsets.md]: Screenshot 2024-09-01 at 1 28 54 PM

This should be rolled back.

File: docs/guidelines/content/set-plagiarism.md
Comment (wescopeland) [inline docs/guidelines/content/set-plagiarism.md]: Please remove this line. Frontmatter at the top of documents is structured data that must conform to an exact format -- this newline could wind up breaking the document at buildtime.

File: docs/es/general/emulator-support-and-issues.md
Comment (wescopeland) [inline docs/es/general/emulator-support-and-issues.md:1]: @gboquizosanchez Not sure if you'd like to weigh in on any of this before I push the button.

**Comment by SuperGamer18** [comment]
Fixed both issues.

File: docs/es/general/emulator-support-and-issues.md
Comment (gboquizosanchez) [inline docs/es/general/emulator-support-and-issues.md]: "No emula Jaguar CD"

File: docs/es/general/emulator-support-and-issues.md
Comment (gboquizosanchez) [inline docs/es/general/emulator-support-and-issues.md]: "Activar renderizado de doble núcleo (aumento de velocidad)"

File: docs/es/general/emulator-support-and-issues.md
Comment (gboquizosanchez) [inline docs/es/general/emulator-support-and-issues.md]: I think this change is not good, we understand "crashear" but I think "bloquear" is better for this context

File: docs/es/general/emulator-support-and-issues.md
Comment (gboquizosanchez) [inline docs/es/general/emulator-support-and-issues.md]: The same than the previous comment

**Review by gboquizosanchez — COMMENTED** [review]
Minor fixes. I think all is properly established except the use of "crash" instead of "bloquear".

**Comment by SuperGamer18** [comment]
Done./Hecho.

File: docs/es/general/emulator-support-and-issues.md
Comment (gboquizosanchez) [inline docs/es/general/emulator-support-and-issues.md]: Sorry, I forget this crash....

Comment (SuperGamer18) [reply docs/es/general/emulator-support-and-issues.md]: 👍

Comment (gboquizosanchez) [reply docs/es/general/emulator-support-and-issues.md]: Thanks! And sorry!

Comment (SuperGamer18) [reply docs/es/general/emulator-support-and-issues.md]: It's ok! We all make mistakes and forget stuff, no matter our age! As you can see in this PR and #83 (the "prequel" to this PR) I definitely made a good amount of mistakes before...

**Review by wescopeland — APPROVED** [review]
Thanks!

**Comment by Jamiras** [comment]
> Speaking as a dev, I only would need to know which cores I could use. If a core wasn't listed as supported, then I would certainly not assume it was viable.
> 
> The page was in dire need of a significant oversimplification and since are interested in cleaning it up, this is a great opportunity to slash and burn all the extra unnecessary info that it had accumulated. Many of the existing docs will benefit from simplification and while it's a slow process, it's definitely an important undertaking.

We still need _somewhere_ to document _why_ certain cores aren't supported. It really helps provide information to pass on to other users when they create tickets against unsupported cores. Additionally, it serves as a reminder the next time we evaluate the cores.

This data is not insignificant. If we want to move it out of this file, fine. But we shouldn't just discard it. I'd be fine with a Unsupported Emulators page under the developer docs.

>   - Appears to work fine, but the core is a bit of a headache to use.
>    - The BIOS has to be configured within the core (F1 > Emulator Configuration > System ROM settings)
>    - Couldn't figure out controls - does the joystick need to be configured within the core too?
>    - Allows loading arbitrary files from the internal menu, which bypasses hashing

The stuff that has dates also lets us track when some was last known to not be working, so if something is working now we can look through history to see if it's obvious why.

>  - 4 Jun 2023 - woz files are not supported
>  - 4 Jun 2023 - joystick not supported. keyboard has to be manually mapped.
>  - 4 Jun 2023 - have to manually create and load a multi-disk bundle for games with multiple disks. disks are generically labelled by index only.

Even just documenting that we haven't looked into a specific core is useful. A core not appearing in the untested list can be assumed to be a new core, which may have significance in itself.

> - ❓ libretro core: **DoubleCherryGB**
> - ❓ libretro core: **Emux GB**
> - ❓ libretro core: **fixGB**
> - ❓ libretro core: **SameBoy**
> - ❓ libretro core: **TGB Dual**

**Comment by wescopeland** [comment]
Not a problem at all. I will get all that metadata restored onto a separate page.

**Pull request merged**
